documentation/wtpdf-from-latex.md

```diff
@@ -148,7 +152,9 @@ This document shows a typical two-column document incorporating tables, figures
 
 Apart from two commands at the start to enable tagging, and a small
 amount of additional markup to give alternative texts for graphics
-inclusion, and to specify the heading rows of tables. The document
+inclusion, and to specify the heading rows of tables\footnote{The%
```

Comment:
percent why?

Reply:
@FrankMittelbach  because I meant to add a % and newline after the `{` but missed by a word.:-) (already fixed n the following commit)

Reply:
Grr it turns out `{%` was a bad idea anyway as it's jekyll syntax annd so i broke the gh-pages build, update coming up